### PR TITLE
add backend community blog post

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: content/en/blog/starting-a-community/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+draft: false
+weight: 50
+images: ["thumbnail.png"]
+contributors: ["Aviram Hassan"]
+---
+
+Hey all, this blog post is a bit meta. tl;dr - we’re starting a [backend developers community on Discord](https://discord.gg/pSKEdmNZcK). Its purpose is to have a place for backend developers to have discussions, share ideas, and collaborate. 
+
+## Why do we need another community?
+Most developer communities are dedicated to languages (Go, Python, Rust) or frameworks (FastAPI, Tokio). However, most modern backend developers work on complex cloud architectures combining multiple languages and ecosystems, and this is only becoming more common as microservice architectures become more prevalent. The modern backend engineer is a [polyglot](https://medium.com/soluto-nashville/why-be-a-polyglot-software-engineer-29cf53465d15), making them more versatile and equipped to solve a wider array of problems. Besides, backend developers face many common issues *unrelated* to the language or ecosystem they use - things to do with architecture, databases and other third party services, CI/CD and SDLC, even development methodology and team management .
```
Comment (infiniteregrets) [inline content/en/blog/starting-a-community/index.md:16]: Small nitpick, extra space before the fullstop

File: content/en/blog/starting-a-community/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Why do we need another community?
+Most developer communities are dedicated to languages (Go, Python, Rust) or frameworks (FastAPI, Tokio). However, most modern backend developers work on complex cloud architectures combining multiple languages and ecosystems, and this is only becoming more common as microservice architectures become more prevalent. The modern backend engineer is a [polyglot](https://medium.com/soluto-nashville/why-be-a-polyglot-software-engineer-29cf53465d15), making them more versatile and equipped to solve a wider array of problems. Besides, backend developers face many common issues *unrelated* to the language or ecosystem they use - things to do with architecture, databases and other third party services, CI/CD and SDLC, even development methodology and team management . 
+
+## What is it going to cover?
+Right now, in addition to having a place to discuss the issues mentioned above, our main ideas are showcasing work, job searching, seeking help and support, sharing tools and of course - posting photos of your pets.
```
Comment (infiniteregrets) [inline content/en/blog/starting-a-community/index.md:19]: "our main ideas are", maybe "include"?

**Review by infiniteregrets — APPROVED** [review]
great! LGTM

**Comment by aviramha** [comment]
Thanks @eyalb181 and @infiniteregrets for feedback!

**Pull request merged**